package-res/resources/web/pentaho/visual/base/View.js

```diff
@@ -68,7 +65,7 @@ define([
        * @protected
        * @readonly
        */
-      this._element = element;
+      this._element = null;
```

Comment:
The visualization can use an HTMLElement or a text node, so we should not call it `_element` anymore, but something like `_domNode`.